client/components/gsuite/gsuite-features/index.jsx

```diff
@@ -1,8 +1,4 @@
-import {
-	GOOGLE_WORKSPACE_BUSINESS_STARTER_YEARLY,
-	GSUITE_BASIC_SLUG,
-	GSUITE_BUSINESS_SLUG,
-} from '@automattic/calypso-products';
+import { GSUITE_BUSINESS_SLUG } from '@automattic/calypso-products';
 import { useTranslate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import googleDocsIcon from 'calypso/assets/images/email-providers/google-workspace/services/docs.svg';
```

Comment:
NIT and Not in your scope of changes! But if you feel it worths, we can replace the line 8 (I can't comment on it) with this:
`import GSuiteSingleFeature from 'calypso/components/gsuite/gsuite-features/single-feature';`

Reply:
Good idea!